client/src/components/Header/Header.js

```diff
@@ -77,6 +79,12 @@ const Header = () => {
             <AuthButton />
           </MenuItem>
         </Menu>
+
```

Comment:
It's not in the `Header` component. It's being shoved up there with absolution css positioning 😅 . Reason is that we don't want the map SearchBar to be on any page besides the initial map page. However, the header is currently the same for all pages. I also think we'll be removing the green header at some point, so this seemed like a fine solution to me for now.

Reply:
Yeah, let's make the `SearchBar` be a truly generic presentational component. Each page should be able to display the it - will just need to have a separate component to manage how to generate the search results and feed it into `SearchBar`, since that behavior differs between data, sources, and map